src/common/BaseAnnotator.ts

```diff
@@ -91,6 +100,15 @@ export default class BaseAnnotator {
         // Called by box-content-preview
     }
 
+    setActiveAnnotationById = (annotationId: string): void => {
+        const annotationExists = getAnnotation(this.store.getState(), annotationId) !== undefined;
+        if (!annotationId || !annotationExists) {
```

Comment:
I think we would still want to set the id in the store even if the annotation isn't yet available. This would allow for delays in startup API calls, for example, and could help avoid race conditions.

Reply:
Yeah, I was thinking if that might be better suited as an option in the Annotator constructor to set the preloaded state for the store. IMO, this API is done by explicit calls or event based triggers by consuming libraries after startup

Reply:
I'm not sure where that leaves us. I think this check is unnecessary and also very likely to lead to subtle bugs in the future.

Reply:
Ok, we'll keep it restriction free, I'll remove the check